static/src/js/components/CollectionDetails/CollectionDetailsBody.js

```diff
@@ -135,13 +135,14 @@ export const CollectionDetailsBody = ({
   const outputFormats = []
   if (services) {
     const { items } = services
-
-    items.forEach((service) => {
-      const { supported_output_formats: formats } = service
-      if (formats) {
-        outputFormats.push(...formats)
-      }
-    })
+    if (items) {
```

Comment:
instead of adding another if, could you default items to `[]` in the destructure above? or `Array.from(items)`?

Reply:
I tried that, it doesn't destructure because the value is `null`, because graph.